lib/display.ts

```diff
@@ -0,0 +1,289 @@
+import { createFromJSON, DepGraph } from "@snyk/dep-graph";
+import chalk from "chalk";
+
+import {
+  BaseImageRemediationAdvice,
+  ContainerTarget,
+  Issue,
+  Options,
+  ScanResult,
+  TestResult,
+} from "./types";
+
+const BREAK_LINE = "\n";
```

Comment:
Can we make this `os.EOL` so that it also works on Windows?

Reply:
Also, let's not forget tests for windows 😬

Reply:
To test for Windows do I need to copy the test file to `test/windows` folder?